components/HomeSection.vue

```diff
@@ -72,7 +73,26 @@ export default Vue.extend({
     };
   },
   computed: mapState<AppState>({
-    items: (state: AppState) => state.homeSection.items
+    items(state: AppState): BaseItemDto[] {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
```

Comment:
Why?

Reply:
I was getting `any Property 'section' does not exist on type '{}'.` errors. Not entirely sure why 🤷‍♀️

Reply:
@Camc314 Fixed them, hope you don't mind!